client.go

```diff
@@ -15,12 +15,13 @@ type Client interface {
 	GetPipelineGroups() ([]*PipelineGroup, error)
 
 	// Pipelines API
-	GetPipelineInstance(string, int) (*PipelineInstance, error)
-	GetPipelineHistoryPage(string, int) (*PipelineHistoryPage, error)
-	GetPipelineStatus(string) (*PipelineStatus, error)
-	PausePipeline(string, string) (*SimpleMessage, error)
-	UnpausePipeline(string) (*SimpleMessage, error)
-	UnlockPipeline(string) (*SimpleMessage, error)
+	PipelineGetInstance(string, int) (*PipelineInstance, error)
```

Comment:
I'm under the impression that methods would generally have to start with an action word. Like Get / Pause / Unlock. Is there a reason we're changing that and moving towards `Pipeline` as a prefix?